src/utils/pagination.ts

```diff
@@ -0,0 +1,34 @@
+import { Request } from 'express'
+
+export const DEFAULT_LIMIT = 10000
+
+export const getPaginationParams = (req: Request) => {
+  const { limit, page } = req.query
+  const parsedLimit = Number(limit)
+  const parsedPage = Number(page)
+  return {
+    limit: limit && !isNaN(parsedLimit) ? parsedLimit : undefined,
+    page: page && !isNaN(parsedPage) ? parsedPage : undefined,
+  }
+}
+
+export const getOffset = (page?: number, limit?: number) => {
+  return limit && page ? limit * (page - 1) : undefined
```

Comment:
Mhmm good question, it's for those cases where the `page` & `limit` are not passed as query params, maybe it can be catch before calling this function. I'll have a look